src/schema/objects/metadata.yaml

```diff
@@ -2287,6 +2397,12 @@ OtherAcquisitionParameters:
   description: |
     Description of other relevant image acquisition parameters.
   type: string
+OuterVolumeSuppression:
+  name: OuterVolumeSuppression
+  display_name: Other-Volume Suppression
```

Comment:
This needs to be corrected to `Outer-Volume Suppression`

Reply:
Corrected in [2c969bb](https://github.com/bids-standard/bids-specification/pull/1377/commits/2c969bb35e448f475d856f8dbf9fff505d67172e)